src/padding_mode.cpp

```diff
@@ -62,43 +55,25 @@ void NoPadding::prepareOpenSSLContext(openssl::SSL_EVP_PKEY_CTX_Ptr& ctx) const
 }
 
 /*
- * PKCSEncryptionPadding
+ * PKCSPadding
  */
 
-void PKCSEncryptionPadding::prepareOpenSSLContext(openssl::SSL_EVP_PKEY_CTX_Ptr& ctx) const
-{
-    _EVP_PKEY_CTX_set_rsa_padding(ctx.get(), static_cast<int>(openssl::RSAPaddingMode::PKCS1));
-}
-
-int PKCSEncryptionPadding::getDataBlockSize(const AsymmetricPublicKey &key) const
+bool PKCSPadding::checkMessageSize(const AsymmetricPublicKey &key, size_t messageSize) const
 {
-    return key.getKeySize()/8 - c_pkcsMaxSizeSOverhead;
+    int remainingSize = key.getKeySize()/8 - c_pkcsMaxSizeSOverhead;
```

Comment:
c_
please remove this weird remnant of hungarian notation. This library doesn't use it. Constants are usually uppercased with underscores. so PKCS_MAX_SIZE_OVERHEAD